src/generic3/values.yaml

```diff
@@ -265,26 +258,20 @@ tolerations: []
 
 affinity: {}
 
+#Redis Subchart activation and overrides
+redis:
+  enabled: false
+  usePassword: false
```

Comment:
Where are you using this override?

Reply:
It overrides `values.yml` in the redis subchart.
Only line 263 is used in generic3 chart